layouts/shortcodes/tooltip.html

```diff
@@ -0,0 +1,47 @@
+<!-- Get shortcode parameters -->
+{{ $text := .Get "text" }}
+{{ $tooltip := .Get "tooltip" }}
+{{ $glossary := .Get "glossary" }}
+
+<!-- Initialize variables -->
+{{ $displayText := $text }}
+{{ $tooltipContent := $tooltip }}
+{{ $fullDefinitionLink := "" }}
+
+<!-- Handle glossary terms if provided -->
+{{ if $glossary }}
+  <!-- Add underscore to multi-word terms to match file name -->
+  {{ $glossaryTerm := replace $glossary " " "_" }}
+  <!-- Get the bundle of glossary term -->
+  {{ $headless := .Site.GetPage "/glossary/terms" }}
+  <!-- Find the matching glossary term file -->
+  {{ with $headless.Resources.GetMatch (printf "%s.md" $glossaryTerm) }}
+    {{ $displayText = .Title }}
+    <!-- Use short definition if available, otherwise use full content -->
+    {{ if .Params.short_definition }}
+      {{ $tooltipContent = .Params.short_definition }}
+      {{ $fullDefinitionLink = printf "/glossary/#%s" ($glossary | urlize) }}
+    {{ else }}
+      <!-- Remove links from full content -->
+      {{ $content := .Content | plainify }}
+      {{ $tooltipContent = $content | replaceRE `\n\[\d+\]:\s+.*` "" | replaceRE `\[([^\]]+)\]\[\d+\]` "$1" | safeHTML }}
+    {{ end }}
+  {{ else }}
+    {{ warnf "Glossary term '%s' not found" $glossary }}
+    {{ $tooltipContent = printf "Glossary term '%s' not found" $glossary }}
+  {{ end }}
+{{ end }}
+
+<!-- Render the tooltip -->
+<span class="tooltip-container" x-data="{ open: false }" @mouseenter="open = true" @mouseleave="open = false">
+  <button class="tooltip-trigger">{{ $displayText }}</button>
+  <span id="tooltip-{{ $displayText | urlize }}" class="tooltip-content" x-bind:class="{ 'show': open }" role="tooltip">
```

Comment:
```suggestion
  <span id="tooltip-{{ $displayText | anchorize }}" class="tooltip-content" x-bind:class="{ 'show': open }" role="tooltip">
```

@brett0000FF I recommend using `anchorize` instead of `urlize` here https://gohugo.io/functions/urls/anchorize/#readout